package.json

```diff
@@ -132,8 +132,8 @@
     "shell-path": "2.1.0"
   },
   "optionalDependencies": {
-    "screen-snippet": "git+https://github.com/symphonyoss/ScreenSnippet.git#v1.0.8",
-    "screen-share-indicator-frame": "https://github.com/symphonyoss/ScreenShareIndicatorFrame.git#v1.0.0",
+    "screen-snippet": "git+https://github.com/mattias-symphony/ScreenSnippet2.git",
```

Comment:
Can we point to a specific version please? We can manage releases in ScreenSnippet2. I'd also prefer if the Project name was something like `WindowsSnippingTool` or along those lines as opposed to just `ScreenSnippet2`

Reply:
As soon as the C/C++ pull request is approved and I can merge it, I will give it version 1.0.0, make a release and point to the main repo instead of to my fork.

Reply:
I am not sure we should change the name - the previous snippet tool was also windows only, so calling the new one ScreenSnippet2 keeps the naming consistent. Also, I am not sure what problems it would cause to change the name of a forked repo. Would we need to merge to master first? I just don't see much benefit, if any, in changing it at this point, and I can see there being a cost to it

Reply:
My thought process is that we should have a proper naming convention :) Not sure why the old project was named `ScreenSnippet` as it's quite generic. I thought we can correct that here.

But, for now, we can keep this.